mesheryctl/internal/cli/root/system/model.go

```diff
@@ -128,10 +129,40 @@ mesheryctl system model list --page 2
 		}
 
 		if len(rows) == 0 {
-			// if no model is found
 			utils.Log.Info("No model(s) found")
-		} else {
+			return nil
+		}
+
+		if cmd.Flags().Changed("page") {
 			utils.PrintToTable(header, rows)
+		} else {
+			paginator := utils.NewPaginator(header, rows)
+			paginator.Render()
+			keysEvents, err := keyboard.GetKeys(10)
+			if err != nil {
+				panic(err)
+			}
+			defer func() {
+				_ = keyboard.Close()
+			}()
+
+			for {
+				event := <-keysEvents
```

Comment:
@zakisk Do we have this paginated navigation for other commands eg: Perf profiles, designs and other resources?
I don't think so.

Please extract this paginated navigation logic and make it reusable across commands.

Reply:
@MUzairS15 pagination login is abstracted into Pagination type.